src/index.ts

```diff
@@ -26,6 +29,7 @@ export class Authenticator {
   _cookieExpirationDays: number;
   _disableCookieDomain: boolean;
   _httpOnly: boolean;
+  _sameSite: SameSite;
```

Comment:
I believe this field also should to be nullable

Reply:
It should be optional rather than nullable I think. The param is `sameSite?: SameSite;` meaning `SameSite | undefined` rather than `SameSite | null`. Will fix

Reply:
Yep, I meant optional, sorry